### PR TITLE
Bar Chart: update demo description

### PR DESCRIPTION
**Pull request by AlexanderTemnov**

### CODE REVIEW

**Review by AnnaKondratova — DISMISSED** [review]

File: JSDemos/Demos/Charts/StandardBar/description.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+
+Assign one or several series to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) property and specify the following settings:
+
+- The series [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type)
+
+- Data source fields
+
+   Assign an approproate field name to the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField) properties if you want to bind a series to data directly. 
+ 
+  Alternatively, you can [use a series template](/Documentation/Guide/UI_Components/Chart/Data_Binding/Bind_Series_to_Data/#Using_a_Series_Template) to bind series to data, if a data source field contains names for series.
```
Comment (arminal) [inline JSDemos/Demos/Charts/StandardBar/description.md]:
```suggestion
  Alternatively, you can [use a series template](/Documentation/Guide/UI_Components/Chart/Data_Binding/Bind_Series_to_Data/#Using_a_Series_Template) to bind series to data if a data source field contains names for series. 
```

File: JSDemos/Demos/Charts/StandardBar/description.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+* [Simple Array](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Local_Array/)
+* [JSON Data](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Read-Only_Data_in_JSON_Format/)
+* [OData Service](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/OData/)
+* [Custom Sources](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Custom_Data_Sources/)
+
+In this demo, the Bar chart is populated with data taken from a simple JavaScript array.
+
+## Configure Series
+
+Assign one or several series to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) property and specify the following settings:
```
Comment (arminal) [inline JSDemos/Demos/Charts/StandardBar/description.md:16]: Please review the way you have listed things below. You have subcategories, so you need to list those differently from how you have done it here. Also, use punctuation when listing in this case.

**Review by arminal — COMMENTED** [review]
#GrammarReview

Please review comma usage before "if". Also, take a look at punctuation when creating lists.

File: JSDemos/Demos/Charts/StandardBar/description.md (hunk 2 above)
Comment (AlexanderTemnov) [reply JSDemos/Demos/Charts/StandardBar/description.md:16]: @arminal  done, thanks.

**Review by AnnaKondratova — DISMISSED** [review]

**Review by AnnaKondratova — APPROVED** [review]

**Pull request merged**